cfy_manager/components/manager/manager.py

```diff
@@ -93,6 +93,23 @@ def _set_selinux_permissive(self):
         else:
             logger.debug('SELinux is not enforced.')
 
+    def _set_selinux_enforcing(self):
+        # Make sure SELinux policies are enforced
+        selinux_state = self._get_selinux_state()
+        logger.debug('Checking whether SELinux in permissive...')
+        if selinux_state == 'Permissive':
+            logger.info('Enforcing SELinux security policy.')
+            common.sudo(['setenforce', 'enforcing'])
+            replace_in_file(
+                'SELINUX=permissive',
+                'SELINUX=enforcing',
+                '/etc/selinux/config')
+            logger.debug('SELinux is enforced.')
+        elif selinux_state == 'Disabled':
+            logger.warning('SELinux is disabled.')
```

Comment:
If we are going to be setting it enforcing rather than leaving it at $user_setup then you should probably do the selinux config file updates here and then log that a reboot is recommended (or cause one to happen somehow at the end of the install process).